pennylane/ops/op_math/controlled.py

```diff
@@ -82,7 +93,39 @@ def circuit(x):
     >>> qml.simplify(qml.adjoint(op))
     Controlled(RY(12.466370614359173, wires=[0]) @ RX(10.166370614359172, wires=[0]), control_wires=[1])
 
+    **Example with compiler**
+
+    .. code-block:: python
+
+        dev = qml.device("lightning.qubit", wires=2)
+
+        @qml.qjit
+        @qml.qnode(dev)
+        def workflow(theta, w, cw):
+            qml.Hadamard(wires=[0])
+            qml.Hadamard(wires=[1])
+
+            def func(arg):
+              qml.RX(theta, wires=arg)
+
+            @qml.cond(theta > 0.0)
+            def cond_fn():
+              qml.RY(theta, wires=w)
+
+            qml.ctrl(func, control=[cw])(w)
+            qml.ctrl(cond_fn, control=[cw])()
+            qml.ctrl(qml.RZ, control=[cw])(theta, wires=w)
+            qml.ctrl(qml.RY(theta, wires=w), control=[cw])
+            return qml.probs()
+
+    >>> workflow(jnp.pi/4, 1, 0)
+    array([0.25, 0.25, 0.03661165, 0.46338835])
     """
+    if active_jit := compiler.active_compiler():
```

Comment:
Minor by I think it's closer to PL's style, right?
```suggestion

    if active_jit := compiler.active_compiler():
```